index.html

```diff
@@ -110,11 +110,15 @@ <h2 class="connect">LET’S CONNECT</h2>
 
     <section id="works">
       <article class="work1 card-effect">
-        <img src="images/work1-img.svg" alt="Work image" class="mobile" />
+        <img
+          src="images/work1-img.svg"
+          alt="Work image"
+          class="mobile workImg"
+        />
```

Comment:
- [ ] I think here you should delete all the works section and create it by the javascript file so according to the requiremnts please add the works section from the javscript file not here 🔥🔥